lib/call-cabal-project-to-nix.nix

```diff
@@ -228,9 +228,9 @@ let
                   then throw "${inputMap.${repoData.url}.rev} may not match ${repoData.ref} for ${repoData.url} use \"${repoData.url}/${repoData.ref}\" as the inputMap key if ${repoData.ref} is a branch or tag that points to ${inputMap.${repoData.url}.rev}."
                   else inputMap.${repoData.url})
             else if repoData.sha256 != null
-              then fetchgit { inherit (repoData) url sha256; rev = repoData.ref; }
+            then fetchgit { inherit (repoData) url sha256 rev; }
```

Comment:
What happens here if we have a `repoData.ref` but no `repoData.rev`?

Reply:
Good question!

Looks like `fetchgit`'s `rev` attribute can either be a commit *or* a `ref`-like attribute, so I should do `rev = repoData.rev or repoData.ref` instead of inheriting.